exercise/fusion.js

```diff
@@ -0,0 +1,106 @@
+/*
+## fusion
+
+### Instructions
+
+The objective of this exercise is to merge objects depending on the values type
+
+Create a function `fusion` that will merge two objects:
+- If the type is an array you must append it
```

Comment:
please say `concat` for arrays too since it's how it's called in JS.
append is very `go` 😉